plugins/decklink/DecklinkInput.cpp

```diff
@@ -81,12 +73,19 @@ bool DeckLink::Activate(DeckLinkDevice *device, long long modeId)
 	if (instance == nullptr)
 		return false;
 
-	DeckLinkDeviceMode *mode = GetDevice()->FindMode(modeId);
+	if (curDevice == nullptr) {
+		LOG(LOG_ERROR, "Tried to activate an input without nullptr device.");
+		return false;
+	}
+
+	DeckLinkDeviceMode *mode = GetDevice()->FindInputMode(modeId);
 	if (mode == nullptr) {
 		instance = nullptr;
 		return false;
 	}
 
+	//instance->setOutputSize();
```

Comment:
If this is just commented out for testing, you'll want to remove this and the following blank line before getting merged.